EventListener/DashboardSubscriber.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * @copyright   2018 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace MauticPlugin\MauticContactSourceBundle\EventListener;
+
+use Mautic\DashboardBundle\Event\WidgetDetailEvent;
+use Mautic\DashboardBundle\EventListener\DashboardSubscriber as MainDashboardSubscriber;
+use MauticPlugin\MauticContactLedgerBundle\Model\LedgerEntryModel;
```

Comment:
This doesn't appear to need/use this model at all... So let's remove all mentions of `entryModel`.